### PR TITLE
mark health check failure at startup

### PR DESCRIPTION
**Pull request by vangork**
set update_at to an earlier time to make sure is_healthy marked false at startup

### CODE REVIEW

File: apps/Cargo.toml [hunk 1]
```diff
@@ -33,9 +33,9 @@ gst-rtsp-server = { package = "gstreamer-rtsp-server", git = "https://gitlab.fre
 gst-sdp = { package = "gstreamer-sdp", git = "https://gitlab.freedesktop.org/gstreamer/gstreamer-rs" }
 gstreamer-video = { git = "https://gitlab.freedesktop.org/gstreamer/gstreamer-rs" }
 gtk = { git = "https://github.com/gtk-rs/gtk-rs" }
-pravega-client = { git = "https://github.com/pravega/pravega-client-rust", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
-pravega-client-config = { git = "https://github.com/pravega/pravega-client-rust", package = "pravega-client-config", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
-pravega-client-shared = { git = "https://github.com/pravega/pravega-client-rust", package = "pravega-client-shared", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
+pravega-client = { git = "https://github.com/pravega/pravega-client-rust", rev = "6614df550bd286e3d1867022c3f235836c6cc278" }
```
Comment (jkhalack) [inline apps/Cargo.toml:36]: How are these changes related to the healthcheck?

File: gst-plugin-pravega/src/pravegasrc/imp.rs [hunk 2]
```diff
@@ -584,6 +584,7 @@ impl BaseSrcImpl for PravegaSrc {
                     ..Default::default()
                 },
                 retention: Default::default(),
+                tags: None,
```
Comment (jkhalack) [inline gst-plugin-pravega/src/pravegasrc/imp.rs:587]: Where is this change gets used?

**Review by claudiofahey — APPROVED** [review]

File: apps/Cargo.toml (hunk 1 above)
Comment (claudiofahey) [reply apps/Cargo.toml:36]: This PR was based on the previous PR. Only the last commit was specifically related to this PR.

Comment (jkhalack) [reply apps/Cargo.toml:36]: Do you mean that we needed to update pravega-client to get the healthcheck working properly?
